Physical_Engine/Query/SolidVolume.cs

```diff
@@ -82,16 +82,34 @@ public static double SolidVolume(this oM.Physical.Elements.ISurface surface)
 
         /***************************************************/
 
-        [Description("Return the total volume of BulkSolids")]
-        [Input("bulkSolids", "Solid geometric elements that have a material composition")]
-        [Output("volume", "The combined volume of BulkSolids", typeof(Volume))]
-        public static double SolidVolume(this BulkSolids bulkSolids)
+        [Description("Return the total volume of SolidBulk.")]
+        [Input("solidBulk", "Solid geometric elements that have a MaterialComposition.")]
+        [Output("volume", "The combined volume of SolidBulk.", typeof(Volume))]
+        public static double SolidVolume(this SolidBulk solidBulk)
         {
-            double solidVolume = bulkSolids.Geometry.Select(x => BH.Engine.Geometry.Query.IVolume(x)).Sum();
+            double solidVolume = solidBulk.Geometry.Select(x => BH.Engine.Geometry.Query.IVolume(x)).Sum();
```

Comment:
Again, I think you are missing null checks on both `solidBulk` and its `Geometry` property.

You could/should also be able to type the volume query as `x.IVolume()`